specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -747,6 +747,143 @@
+          "type": "string"
+        },
+        "maxRetries": {
+          "description": "maximum number of retries to attempt",
+          "type": "string"
+        },
+        "maxInterval": {
+          "description": "determines the maximum interval between retries",
+          "type": "string"
+        }
+      }
+    },
+    "DaprResiliencyCircuitBreakerPoliciesCollection": {
+      "description": "Dapr Resiliency Circuit Breaker Policy ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaprResiliencyCircuitBreakerPolicy"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DaprResiliencyCircuitBreakerPolicy": {
+      "description": "Resiliency Policy Breakers Policy",
+      "type": "object",
+      "properties": {
+        "maxRequests": {
+          "description": "maximum number of requests allowed to pass through",
+          "type": "string"
+        },
+        "interval": {
+          "description": "cyclical period of time used by the CB to clear its internal counts",
+          "type": "string"
+        },
+        "timeout": {
+          "description": "the period of the open state until the CB switches to half-open",
+          "type": "string"
+        },
+        "trip": {
+          "description": "A Common Expression Language (CEL) statement that is evaluated by the CB When the statement evaluates to true, the CB trips and becomes open",
+          "type": "string"
+        },
+        "circuitBreakerScope": {
+          "description": "Specify whether circuit breaking state should be scoped to an individual actor ID, all actors across the actor type, or both. Possible values include id, type, or both",
+          "type": "string"
+        },
+        "circuitBreakerCacheSize": {
+          "description": "Specify a cache size for the number of CBs to keep in memory. The value should be larger than the expected number of active actor instances.",
+          "type": "string"
+        }
+      }
+    },
+    "DaprResiliencyTimeoutPoliciesCollection": {
+      "description": "Dapr Resiliency Timeout Policy ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaprResiliencyTimeoutPolicy"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DaprResiliencyTimeoutPolicy": {
```

Comment:
>DaprResiliencyTimeoutPolicy [](http://example.com/codeflow?start=5&length=27)

needs the ARM envelope. same as the above resource.

Reply:
Added